client/blocks/eligibility-warnings/index.tsx

```diff
@@ -89,6 +89,7 @@ export const EligibilityWarnings = ( {
 			'eligibility-warnings__placeholder': isPlaceholder,
 			'eligibility-warnings--with-indent': showWarnings,
 			'eligibility-warnings--blocking-hold': hasBlockingHold( listHolds ),
+			'eligibility-warnings--without-title': ! title && ! primaryText,
```

Comment:
If we don't have a `title` and `primaryText` this component will add a special padding-top to scenarios like this:

![image](https://user-images.githubusercontent.com/1044309/193106325-f5c4b43d-039d-4f3f-92e5-9cdc20a5c0e8.png)